russell/block-templates/header-footer-only.html

```diff
@@ -7,4 +7,8 @@
 </main>
 <!-- /wp:group -->
 
-<!-- wp:template-part {"slug":"footer","tagName":"footer","layout":{"inherit":true},"className":"site-footer-container"} /-->
+<!-- wp:group {"layout":{"inherit":true}} -->
```

Comment:
None of the other templates for Payton have this, so I think we should just remove it for consistency.